mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java

```diff
@@ -42,7 +42,7 @@ public MilStd2525SinglePoint(MapInstance mapInstance, IMilStdRenderer iconRender
         this.placemark = new EMPPlacemark(this, position);
         this.sSymbolCode = symbol.getSymbolCode();
         this.oRenderer = iconRenderer;
-        this.setSymbolAttributes();
+        this.oAttributes = symbol.getAttributes();
```

Comment:
To me this is the most concerning change. Please review carefully.

Reply:
So the user has to do a symbol.setAttributes or something like that first?

Reply:
Now the user can simple call symbol.setIconColor(color). They also have the ability to call symbol.getAttributes and put other attributes there if they know the key value.

Reply:
attributes is created at object creation time for symbols so no setting is required for that.

Reply:
great!